using_images/db_images/mongodb.adoc

```diff
@@ -178,48 +200,51 @@ method to change passwords for the database user (`*MONGODB_USER*`) and *admin*
 user is by changing the environment variables `*MONGODB_PASSWORD*` and
 `*MONGODB_ADMIN_PASSWORD*`, respectively.
 
+You can view the current passwords by viewing the pod in the web console or by
+listing the environment variables:
+
+----
+$ oc env pod/<pod_name> --list
+----
+
 Changing database passwords directly in MongoDB causes a mismatch between the
 values stored in the variables and the actual passwords. Whenever a database
 container starts, it resets the passwords to the values stored in the
 environment variables.
 
-You can change these passwords with:
+To change these passwords, update one or both of the desired environment
+variables for the related deployment configuration(s) using the `oc env`
+command. If multiple deployment configurations utilize these environment
+variables, for example in the case of an application created from a template,
+you must update the variables on each deployment configuration so that the
+passwords are in sync everywhere. This can be done all in the same command:
 
 ----
-$ oc env dc mongodb \
+$ oc env dc <dc_name> [<dc_name_2> ...] \
   MONGODB_PASSWORD=<new_password> \
   MONGODB_ADMIN_PASSWORD=<new_admin_password>
```

Comment:
@rhcarvalho Updated paragraph + cmd per suggestions (here and in MySQL). :point_up: WDYT?